htmlelements-java/src/main/java/ru/yandex/qatools/htmlelements/element/TypifiedElement.java

```diff
@@ -91,9 +91,15 @@ public String toString() {
      * @return Whether or not the element is displayed
      */
     public boolean isDisplayed() {
-        return getWrappedElement().isDisplayed();
+        try{
+            return getWrappedElement().isDisplayed();
+        } catch (Exception e) {
+            //ignore and return false to indicate element is not visible.
+        }
+        return false;
```

Comment:
Here you're presuming that if element isn't found (this is when exception is thrown), then it's invisible. But this is hiding that fact (that element wasn't found) from the developer who runs the tests and therefore I think it's not the right way to handle this.

Instead you should be catching that exception in place where you call `isDisplayed` method.